cfme/utils/appliance/db.py

```diff
@@ -263,6 +266,43 @@ def loosen_pgssl(self, with_ssl=False):
         result = client.run_command("systemctl restart {scl}-postgresql".format(scl=scl))
         return result.rc
 
+    def create_db_lvm(self):
+        """
+        Set up a partition for the CFME DB to run on.
+
+        As a work-around for having to provide a separate disk to a CFME appliance
+        for the database, we instead partition the single disk we have and run
+        the DB on the new partition.
+
+        This requires that the appliance's disk has more space than CFME requires.
+        For example, on RHOS, downstream CFME uses 43GB on a disk but the flavor used
+        to deploy the template has a 75GB disk. Therefore, we have extra space which
+        we can partition.
+
+        Note that this is not the 'ideal' way of doing things and should
+        be a stop-gap measure until we are capabale of attaching additional disks to
+        an appliance via automation on all infra types.
+        """
+        self.logger.info("Creating new LVM for db using remaining space on /dev/vda...")
+
+        fstab_line = '/dev/mapper/dbvg-dblv $APPLIANCE_PG_MOUNT_POINT xfs defaults 0 0'
```

Comment:
QUESTION: is this variable always present $APPLIANCE_PG_MOUNT_POINT in appliance ?
where is it defined ?

Reply:
Yes it is, it is an env variable, talked about in this doc: https://access.redhat.com/documentation/en-us/red_hat_cloudforms/4.6/html/installing_red_hat_cloudforms_on_google_compute_engine/configuring-cloudforms#configuring-an-internal-database